requirements/requirements.txt

```diff
@@ -2,7 +2,7 @@
 # This file is autogenerated by pip-compile with Python 3.11
 # by the following command:
 #
-#    inv requirements
+#    pip-compile requirements/requirements.in
```

Comment:
out of curiosity, did you use `inv requirements` or `pip-compile requirements/requirements.in`?

Reply:
no, just raw `pip-compile`. For some reason `inv requirements` didn't want to generate a requirements.txt file for me, so I resorted to the snail. I forgot about this actually. Let me try again